letsencrypt/auth_handler.py

```diff
@@ -113,6 +113,13 @@ def _solve_challenges(self):
                     cont_resp = self.cont_auth.perform(self.cont_c)
                 if self.dv_c:
                     dv_resp = self.dv_auth.perform(self.dv_c)
+                    for response in dv_resp:
+                        for achall in self.dv_c:
+                            if not response.simple_verify(
```

Comment:
That will not work, because `simple_verify` takes different arguments depending on the response class. You have to condition on `isinstance`